main.tf

```diff
@@ -288,7 +289,7 @@ resource "aci_rest_managed" "l3extVirtualLIfP" {
   class_name = "l3extVirtualLIfP"
   content = {
     addr       = each.value.ip
-    autostate  = "disabled"
+    autostate  = each.value.autostate ? "enabled" : "disabled"
```

Comment:
We need to add the `autostate` value and do the conversion under `local.floating_interfaces` first, and then we can use it here.